inspire_json_merger/api.py

```diff
@@ -138,3 +142,15 @@ def get_acquisition_source(json_obj):
 
 def is_manual_merge(head, update):
     return ('control_number' in update and 'control_number' in head and update['control_number'] != head['control_number'])
+
+
+def is_erratum(update):
+    erratum_keywords = {"erratum", "corrigendum", "publisher's note"}
+    journal_titles_list = get_value(update, "titles.title", [])
+    journal_titles_string = " ".join(journal_titles_list).lower()
+    if any([keyword in journal_titles_string for keyword in erratum_keywords]):
+        return True
+    if any(journal_title.lower().startswith('correction to:') for journal_title in journal_titles_list):
+        return True
+    if 'erratum' in get_value(update, "dois.material", []):
+        return True
```

Comment:
maybe name the clauses like `title_contains_erratum_keyword`, etc., so you can simply `return ... or ... or ...` ?